scheduler/scheduler_dfs.go

```diff
@@ -193,14 +194,26 @@ func (s *syncClient) resolveResourcesDfs(ctx context.Context, table *schema.Tabl
 		resolvedResources <- resource
 		for _, relation := range resource.Table.Relations {
 			relation := relation
+			tableConcurrencyKey := table.Name + "-" + client.ID()
```

Comment:
Unless I'm missing something, it seems like right now the `single table max concurrency` setting only applies to relational (ie. child) tables, and not top-level ones. As the name implies it would apply to all tables, I think we should either change the name or make it apply at the top-level as well.

My vote would go for making it apply to all tables. For top-level tables, it could still be useful. Imagine you have concurrency of 100, 10 tables and 100 clients. Without a per-table limit, the DFS scheduler would assign all 100 slots to the first table, which might not be desirable, so having a lever there would be useful.

Reply:
But at the top level aren't all top level tables a unique pair of `table.Name` and `client.ID`?